assets/js/search.js

```diff
@@ -0,0 +1,7 @@
+//doesn't allow to search if textfield is empty
+  $(document).on('mouseover', '#search_form', function(){
+    if($('#search_field').val() !== '')
```

Comment:
this is wrong,  if else statement should expected to have {} to determined code blocks

 e.g: if (foo){
} else {
 pass
}

Reply:
Updated